src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -1445,7 +1445,7 @@ public BundleContext getBundleContext()
      * {@inheritDoc}
      */
     @Override
-    public DtlsControlImpl getDtlsControl(Channel channel)
+    public DtlsControlImpl getSrtpControl(Channel channel)
```

Comment:
i think having the method be called `getSrtpControl` but returning `DtlsControlImpl` is confusing, but i get what you're saying.  But, why do we need to cast at all?  this seems to defeat a lot of the purpose of having a class hierarchy (where the behavior differs but the API is the same, so the caller doesn't have to know/care which underlying subclass it's actually dealing with, only that it implements the SrtpControl interface).